settings.js

```diff
@@ -9,24 +9,26 @@ const mkdirp = require('mkdirp');
 
 var {remote} = require('electron');
 
-
 module.exports.dataFolder = path.join(remote.app.getPath("appData"), "Revolutionary-Games",
                                       "Launcher");
 
 module.exports.tmpDLFolder = path.join(remote.app.getPath("temp"),
                                        "Revolutionary-Games-Launcher");
 
-module.exports.installPath = path.join(module.exports.dataFolder, "Installed");
-
 module.exports.locallyCachedDLFile = path.join(module.exports.dataFolder,
                                                "saved_version_db_v2.json");
 
+module.exports.defaultInstallPath = path.join(module.exports.dataFolder, "Installed");
 
 // Make sure it exists. This simplifies a lot of code
 mkdirp.sync(module.exports.dataFolder);
 
+mkdirp.sync(module.exports.defaultInstallPath);
```

Comment:
The user likely won't appreciate unnecessary folders being created, especially if they change the install path.
Code that breaks without the install path existing should catch that as an error (other than the version downloading code which should be the only )

Reply:
Hmm... I thought that was a bug.